Index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>MetaKart</title>
+    <link rel="stylesheet" href="https://hardcore-aryabhata-201d65.netlify.app/Css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Navbar -->
+    <nav class="top-navbar">
+        <div class="top-navbar-icon">
+            <img src="https://bit.ly/3v2WZoY"/>
+            <div class="title">MetaKart</div>
+        </div>
+            <div class="single">
+                <label for="search" style="color: white;  padding: .5rem; " class="fa fa-search fa-2x"></label>
+                <input type="search" name="search" id="search1" placeholder="Search">
+            </div>
+        
+        <div class="top-navbar-menu">
+            <a href="signin.html" class="top-menu home"><button class="btn">Login</button></a> &nbsp;&nbsp;&nbsp;
+            <a class="top-menu" href="cart.html"><i class="fa fa-shopping-cart fa-2x"><span>2</span></i></a> &nbsp;&nbsp;
+            <a  class="top-menu" href=""><i class="fa fa-heart fa-2x"><span>4</span></i></a>
+            &nbsp;&nbsp;&nbsp;            
+            &nbsp;&nbsp;&nbsp;
```

Comment:
Any particular reason why you have used HTML entity here `&nbsp;`. Maybe you can do this flex as well.

Reply:
No Mahesh there isn't any specific reason for that..., Yepp i can use flex for the same but this one i found easy that's why gone through it 😅
Thanks, I will do with flex while implementing media queries....